superset-frontend/src/setup/setupFormatters.ts

```diff
@@ -62,6 +63,105 @@ export default function setupFormatters() {
     .registerValue('$,0', getNumberFormatter('$,.4f'))
     .registerValue('$,0f', getNumberFormatter('$,.4f'))
     .registerValue('$,.f', getNumberFormatter('$,.4f'))
+    .registerValue('฿,.0f',
+      createD3NumberFormatter({
+        formatString: '$,.0f',
```

Comment:
Can we reduce the duplication of this locale stuff at all? Since all that seems to change is the currency symbol?

Reply:
Yeah, working on that right now.